AppCenterDataStorage/AppCenterDataStorageTests/MSTokenExchangeTests.m

```diff
@@ -214,7 +214,7 @@ - (void)testExchangeServiceSerializationFails {
                                                     partition:mockPartition
                                             completionHandler:^(MSTokensResponse *__unused tokenResponses, NSError *_Nullable returnError) {
                                               XCTAssertNotNil(returnError);
-                                              XCTAssertEqual(returnError.code, MSDataStoreErrorJSONSerializationFailed);
+                                              XCTAssertEqual(returnError.code, kMSACDataStoreErrorJSONSerializationFailed);
```

Comment:
Sorry last time I made a mistake on the format for this one. You don't need the `k` because it's from an enum. I didn't realized it was from an enum at that time :confounded:.